front/app/services/projectAllowedInputTopics.ts

```diff
@@ -24,7 +24,7 @@ export interface IProjectAllowedInputTopic {
   };
 }
 
-export interface IProjectAllowedInputTopicsResponse {
+export interface IProjectAllowedInputTopicsData {
```

Comment:
I named this `...Response` because in other places of the application `...Data` is reserved for the `data` attribute of the response. E.g. `front/app/services/events.ts`:

```
export interface IEventData {
  id: string;
  type: string;
  attributes: { ... };
  relationships: { ... };
}

export interface IEvent {
  data: IEventData;
}

export interface IEvents {
  data: IEventData[];
  links: ILinks;
}
```

Maybe a better name would be `IProjectAllowedInputTopics`, for the sake of consistency?